service/src/test/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderServiceTest.java

```diff
@@ -302,6 +303,25 @@ void shouldReturnFormattedExpiryDateWhenInterimSubtypeAndNamedDateSelected(Gener
             .isEqualTo(formatLocalDateToString(time.now().toLocalDate(), "'11:59pm', d MMMM y"));
     }
 
+    @ParameterizedTest
+    @EnumSource(value = GeneratedOrderType.class, names = {"CARE_ORDER", "SUPERVISION_ORDER"})
+    void shouldReturnFormattedExpiryDateWhenInterimSubtypeAndSpecificTimeNamedDateSelected(GeneratedOrderType type) {
+        LocalDateTime localDateTime = LocalDateTime.of(2022, 9, 22, 11, 59);
+        final OrderTypeAndDocument orderTypeAndDocument = orderTypeAndDocument(type, INTERIM);
+        final JudgeAndLegalAdvisor judgeAndLegalAdvisor = testJudgeAndLegalAdviser();
+        final CaseData caseData = caseData()
+            .interimEndDate(InterimEndDate.builder()
+                .type(SPECIFIC_TIME_NAMED_DATE)
+                .endDateTime(localDateTime)
+                .build())
+            .build();
+
+        GeneratedOrder builtOrder = service.buildCompleteOrder(orderTypeAndDocument, judgeAndLegalAdvisor, caseData);
+
+        assertThat(builtOrder.getExpiryDate())
+            .isEqualTo(formatLocalDateTimeBaseUsingFormat(localDateTime, "'11:59am', d MMMM y"));
```

Comment:
Think this would be better as
```suggestion
            .isEqualTo("11:59am, 22 September 2022");
```
or just use the format `"h:mma, d MMMM y"` but with a static date you should know what the output will be

Reply:
cool, will change :)